HyperV/scripts/create-environment.ps1

```diff
@@ -30,9 +30,9 @@ $hasQemuImg = Test-Path $binDir\qemu-img.exe
 
 $pip_conf_content = @"
 [global]
-index-url = http://dl.openstack.tld:8080/cloudbase/CI/+simple/
+index-url = http://10.0.110.1:8080/cloudbase/CI/+simple/
```

Comment:
Shouldn't we use  configuration.ps1 and download location here? And the up address also from configuration.ps1 below?